src/vm/readytoruninfo.cpp

```diff
@@ -334,6 +334,12 @@ PTR_BYTE ReadyToRunInfo::GetDebugInfo(PTR_RUNTIME_FUNCTION pRuntimeFunction)
     return dac_cast<PTR_BYTE>(m_pLayout->GetBase()) + debugInfoOffset;
 }
 
+BOOL ReadyToRunInfo::IsReadyToRunEntryPoint(PCODE pCode)
+{
+    return pCode >= dac_cast<TADDR>(m_pLayout->GetBase()) && 
+           pCode < dac_cast<TADDR>(m_pLayout->GetBase()) + m_pLayout->GetSize();
```

Comment:
It would have been more ideal to use `GetBase + m_pRuntimeFunctions[0].BeginAddress` and `GetBase + m_pRuntimeFunctions[m_nRuntimeFunctions - 1].EndAddress`, but `EndAddress` is not available on x86